conseil-common/src/main/scala/tech/cryptonomic/conseil/common/generic/chain/DataTypes.scala

```diff
@@ -255,7 +259,9 @@ object DataTypes {
       aggregation: List[Aggregation] = List.empty,
       temporalPartition: Option[String] = None,
       snapshot: Option[Snapshot] = None
-  )
+  ) {
+    def adjustLimit(value: Int): Query = copy(limit = Math.min(limit, value))
```

Comment:
I'd suggest a more explicit name like
`capResultsLimit` `withLimitCap`, or similar

"adjust" is very vague

Reply:
Renamed to `withLimitCap`. Does makes more sense that what I suggested